ext/QuantumToolboxCairoMakieExt.jl

```diff
@@ -0,0 +1,215 @@
+module QuantumToolboxCairoMakieExt
+
+using QuantumToolbox
+using CairoMakie
```

Comment:
ah, one more thing

could you list all the functions and names you used in `CairoMakie` ?
That is, instead of `using CairoMakie`, 

```julia
import CairoMakie: heatmap!, ...
```

This allows a more convenient maintenance in the future.